### DISCUSSION TITLE
Border with image

### DISCUSSION THREAD

**Discussion by cordobadigitalmedia** [tailwindlabs/tailwindcss#4401] [Help] [answered]
Hi

I would really appreciate any suggestions on setting up a border with a background image or svg like in this case, with tailwind
https://codepen.io/chris_tudor/pen/OJPeQgb

Thanks

**Comment by olets** [comment, marked as answer]
To convert this exact thing to Tailwind, go through each class and replace it with the relevant utility classes.

For example, `<div class="body …` becomes something like

```
<div class="h-full flex justify-center flex-col bg-140% bg-no-repeat bg-center small-tablet:bg-148vh small-tablet:bg-50%-31vh …
```

`bg-50%-31vh`, `bg-140%`, `bg-148vh`, and `small-tablet` require custom configuration, and could have any names you want.

https://play.tailwindcss.com/WnivgZ5VIF

See
- https://tailwindcss.com/docs/background-position
- https://tailwindcss.com/docs/background-size
- https://tailwindcss.com/docs/background-repeat
- https://tailwindcss.com/docs/flex
- https://tailwindcss.com/docs/flex-direction
- https://tailwindcss.com/docs/height
- https://tailwindcss.com/docs/responsive-design

Note that a basic inset border can be handled without customization: https://play.tailwindcss.com/nCIxxwfIVm

See
- https://tailwindcss.com/docs/ring-width
- https://tailwindcss.com/docs/ring-color
- https://tailwindcss.com/docs/ring-offset-width
- https://tailwindcss.com/docs/ring-offset-color

**Comment by cordobadigitalmedia** [comment]
Thanks for that
